src/sqlfluff/core/rules/base.py

```diff
@@ -249,7 +249,6 @@ def crawl(
         siblings_pre=None,
         siblings_post=None,
         raw_stack=None,
-        fix=False,
         memory=None,

```

Comment:
I was initially skeptical about removing this... but good catch in seeing that nothing is using it! 👍